pandas/core/indexes/base.py

```diff
@@ -2788,26 +2788,55 @@ def set_value(self, arr, key, value):
 
     def _get_level_values(self, level):
         """
+        Return an Index of values for requested level.
+
         Return an Index of values for requested level, equal to the length
         of the index.
 
         Parameters
         ----------
         level : int or str
-            ``level`` is either the integer position of the level in the
-            MultiIndex, or the name of the level.
+            It is either the integer position of the level in the
+            MultiIndex, or the `name` of the level.
```

Comment:
Don't think name has to be in backticks, since it isn't a parameter

Reply:
Since MultiIndex has its own docstring, you probably don't have to reference it here.